### ISSUE TITLE
//regen command doesn't support datapack's generator

### ISSUE THREAD

**Issue by MaxIsV** [IntellectualSites/FastAsyncWorldEdit#1521]
### Server Implementation

Purpur

### Server Version

1.18.1

### Describe the bug

**//regen** command in FAWE doesn't consider that the world was generated using a datapack (Terralith 2.0.9). And the regenerated area is not suitable. This isn't observed in WE 7.2.8

### To Reproduce

1. Select chunk with a **//chunk**
2. Type **//regen**

### Expected behaviour

Regenerated chunk must fit the surrounding area

### Screenshots / Videos

FAWE:
![2022-01-06_01 13 45](https://user-images.githubusercontent.com/58142896/148355279-2b31ef5c-d037-45ea-9155-8246b73a123d.png)

WE:
![2022-01-06_10 42 33](https://user-images.githubusercontent.com/58142896/148355287-c874cc77-4c01-41c4-a80e-354c142771af.png)

### Fawe Debugpaste

https://athion.net/ISPaster/paste/view/6d16b3ae84d1406497a45a21d6dbd007

### Fawe Version

2.0.0-SNAPSHOT-56;59d4247

### Checklist

- [X] I have included a Fawe debugpaste.
- [X] I am using the newest build from https://ci.athion.net/job/FastAsyncWorldEdit/ and the issue still persists.

**Comment by NotMyFault** [comment]
Can you provide a download link to the datapack please?

**Comment by MaxIsV** [comment]
> Can you provide a download link to the datapack please?

https://www.planetminecraft.com/data-pack/terralith-overworld-evolved-100-biomes-caves-and-more/

**Comment by NotMyFault** [comment]
I'm unable to replicate that on Paper.

**Comment by MaxIsV** [comment]
> I'm unable to replicate that on Paper.

To use this datapack, you first need to make a single player world and then move it to the server. But the datapack needs to be added exactly when creating the world.

**Comment by MaxIsV** [comment]
> I'm unable to replicate that on Paper.

Or do you mean that your area is regenerated correctly according to the datapack?

**Comment by NotMyFault** [comment]
> > I'm unable to replicate that on Paper.
> 
> Or do you mean that your area is regenerated correctly according to the datapack?

Yes. I have had no issues trying to regenerate a large part of the world. Everything has been restored like it used to look before I removed it. I was testing it on Paper build No. 155 and Fawe build No. 71.
The world has been generated with your datapack provided.
